Gems/ROS2/Code/Source/VehicleDynamics/Utilities.h

```diff
@@ -67,4 +69,11 @@ namespace ROS2::VehicleDynamics::Utilities
     //! @returns ramped velocity according to time, acceleration and clamped to @param maxVelocity
     float ComputeRampVelocity(float targetVelocity, float lastVelocity, AZ::u64 deltaTimeNs, float acceleration, float maxVelocity);
 
+
+    void SetWheelRotationSpeed(const  VehicleDynamics::WheelDynamicsData& data, float wheelRotationSpeed);
```

Comment:
Please use `clang-format` to format the code
```suggestion
    void SetWheelRotationSpeed(const VehicleDynamics::WheelDynamicsData& data, float wheelRotationSpeed);
```